sdcm/cluster.py

```diff
@@ -2949,6 +2961,15 @@ def tags(self) -> Dict[str, str]:
                 "NodeType": str(self.node_type),
                 "keep_action": "terminate" if action == "destroy" else "", }
 
+    @property
+    def all_nodes_scylla_shards(self):
+        return self._nodes_shards
+
+    def add_node_to_shards_list(self, node: BaseNode):
```

Comment:
remove

Reply:
implement preparation of the dict where the email is generated.

Reply:
moved to the tester.py, were the email data is preparing